CMakeLists.txt

```diff
@@ -625,6 +625,7 @@ endif()
 set(Boost_USE_MULTITHREADED ON)
 # require minimally the bundled version
 if(WITH_SYSTEM_BOOST)
+  message(STATUS "BUILDING Boost Libraries, WITH_SYSTEM_BOOST is set to ON")
```

Comment:
this one seems a bit misleading? if we're using system boost libraries, we're not technically _building_ them

Reply:
ack, reworded